packages/world/src/WorldFactory.sol

```diff
@@ -8,7 +8,9 @@ import { IBaseWorld } from "./codegen/interfaces/IBaseWorld.sol";
 import { IModule } from "./IModule.sol";
 import { ROOT_NAMESPACE_ID } from "./constants.sol";
 
-/**
+import { Test, console } from "forge-std/Test.sol";
```

Comment:
Do we need these imports?

Reply:
You're right, that was on accident. Fixed.